File: ROWNANIE_rownanie_kwadratowe/rownanie.py
import sys
def ileRozwiazan(a,b,c):
    delta = b**2-4*a*c
    if delta > 0:
        return 2
    elif delta ==0:
        return 1
    else:
        return 0

for line in sys.stdin:
    a,b,c = list(map(float, line.split()))
    print(ileRozwiazan(a,b,c))

File: ROWNANIE_rownanie_kwadratowe/rownanie.rb
while line=gets
    a,b,c=line.split().map(&:to_f)
    delta = b**2+4*a*c 
    if delta>0
        print "2\n"
    elsif delta == 0 
        print "1\n"
    else 
        print "0\n"
    end
end